### ISSUE TITLE
about it seems to learn nothing

### ISSUE THREAD

**Issue by sun-sc** [voxelmorph/voxelmorph#408]
hello,  thank you so much for sharing your great work! I have met some questions.
**Task** (what are you trying to do/register?)

I'm using my own data about abdominal CT. I want to registerdifferent ct images of the same patient to account for positional changes caused by respiration. But the result seems the same to the original image.

**What have you tried**

I have already change the data in range(0,1).with mse loss , I use the default model and train 450 epoch.
the loss result are follows:
 loss: 0.0012 - transformer_loss: 9.8923e-04 - flow_resize_loss: 0.0181

the input and output are as follows:

the moving image
![图片](https://user-images.githubusercontent.com/55624370/160268637-1bea5a83-3fae-4fae-82ca-79edd5e37c65.png)
the fixed image
![图片](https://user-images.githubusercontent.com/55624370/160268646-4144dd61-75f4-4d75-90d2-1057a5d18159.png)
the result
![图片](https://user-images.githubusercontent.com/55624370/160268654-38dc57b7-faa0-4aec-802a-5027a55c83ba.png)

you can see that the result looks same to the moving image(such as in the red box).

hope for your help.thank you!

**Comment by adalca** [comment]
can you tell me how large your images are and what the hyperparameter value is?

**Comment by sun-sc** [comment]
thanks for your reply. 
1. the size of one image is 128 * 240 * 288.  
2. batch_size is 1
3. i use 120 pairs to train( one fixed image and one moving image) , and test on 30 pairs
4. the encoder filters is 16 32 32 32
5.  the decorder filters is 32 32 32 32 32 16 16
6. learing rate is 1e-4
7. the loss is mse loss 
8. step per epoch is 100 ,image sigma is 1
(hyperparameter are used the default in ./scripts/tf/train.py , and i try to change the learing rate to 5*1e-4 and ncc loss ，image sigma 0.05 ,respectively， but the result seems the same to the moving image)

hope for your help.thank you!

**Comment by adalca** [comment]
we haven't really used image sigma in a long time, i guess my question is what the size of `lambda` is when using mse? To me if the result looks the same as the moving image, the immediate thing to look at is whether your deformation regularization (`lambda`) is too large.

**Comment by sun-sc** [comment]
hello , the lambda is  [1, 0.01] ( I print the list in `./scripts/tf/train.py  line 178,about the var weights ` , Did I misunderstand what you meant? ）,will i reduct the value of ‘1’ ，such as 0.5, 0.01?

**Comment by adalca** [comment]
no, this seems right. Seems a bit weird that you are getting no deformation fields, though. I would try lowering the 0.01 part to even lower, say 0.001, and ret-training, although this result surprises me. 

You might want to look at [hypermorph](http://hypermorph.voxelmorph.net/), @ahoopes 's recent work.

**Comment by sun-sc** [comment]
I have solved it. Thank you very much! I couldn’t have done it without you!!
